### PR TITLE
Added a bunch of games which work

### PR DESCRIPTION
**Pull request by sharkwouter**
Most of them aren't really popular games, but they work very well.

### CODE REVIEW

File: proton-whitelist.vdf [hunk 1]
```diff
@@ -13,3 +13,54 @@
...
+{
+	"name"		"proton_411"
+}
+
+"520720" // Dear Esther: Landmark Edition
+{
+	"name"		"proton_411"
+}
+
+"321040" // DiRT 3
```
Comment (alkazar) [inline proton-whitelist.vdf]: should be "DiRT 3 Complete Edition"

Comment (sharkwouter) [reply proton-whitelist.vdf]: Fixed

File: proton-whitelist.vdf [hunk 2]
```diff
@@ -13,3 +13,73 @@
...
+{
+	"name"		"proton_411"
+}
+
+"220860" // McPixel
+{
+	"name"		"proton_411"
+}
+
+"32460" // Monkey Island 2 Special Edition: LeChuck’s Revenge
```
Comment (alkazar) [inline proton-whitelist.vdf]: Isn't this already whitelisted by Valve?

Unless updating to a newer version of proton gives some kind of benefit I think we should leave Valve whitelisted games as is.

Comment (sharkwouter) [reply proton-whitelist.vdf]: It seems you are correct. I hadn't noticed.

Comment (sharkwouter) [reply proton-whitelist.vdf]: I've removed Monkey Island 2 now

**Pull request merged**